src/main/java/gov/cabinetoffice/gap/adminbackend/repositories/SubmissionRepository.java

```diff
@@ -26,9 +26,16 @@ public interface SubmissionRepository extends JpaRepository<Submission, UUID> {
 
     @Transactional
     @Modifying
-    @Query(value = "UPDATE grant_submission SET last_required_checks_export = :lastRequiredChecksExport WHERE application_id = :applicationId",
-            nativeQuery = true)
-    void updateSubmissionLastRequiredChecksExportByGrantApplicationId(@Param("applicationId") Integer applicationId,
-            @Param("lastRequiredChecksExport") Instant lastRequiredChecksExport);
+    @Query("""
```

Comment:
Why are all these quotes here? More curious than anything!

Reply:
I used JPA palette for the query and it adds them for some reason but I'll remove it